astropy/visualization/stretch.py

```diff
@@ -80,21 +85,31 @@ def inverse(self):
 
 class LinearStretch(BaseStretch):
     """
-    A linear stretch.
+    A linear stretch with a slope and offset.
 
     The stretch is given by:
 
     .. math::
-        y = x
+        y = ax + b
     """
 
+    def __init__(self, a=1, b=0):
```

Comment:
Could you rename a and b to ``slope`` and ``intercept``? It's good practice to use the same names here as you use for storing the attributes in the class, and also this will then be consistent with the ``Linear1D`` model in astropy.modeling (http://docs.astropy.org/en/stable/api/astropy.modeling.functional_models.Linear1D.html#astropy.modeling.functional_models.Linear1D)

Reply:
Sure. I assume that the power/exp stretches are keeping 'a' for their argument names to not break backwards compatibility?